googleads/ad_manager.py

```diff
@@ -877,12 +877,14 @@ def _GetPqlService(self):
           'PublisherQueryLanguageService', self._version, self._server)
     return self._pql_service
 
-  def WaitForReport(self, report_job):
+  def WaitForReport(self, report_job, pollTimeSeconds = 30):
     """Runs a report, then waits (blocks) for the report to finish generating.
 
     Args:
       report_job: The report job to wait for. This may be a dictionary or an
           instance of the SOAP ReportJob class.
+      poolTimeSeconds: The number of seconds to wait for the report to finish
```

Comment:
pollTimeSeconds: The number of seconds to wait between polling calls to check the report status.